execution-core/src/stake.rs

```diff
@@ -197,8 +197,41 @@ impl Withdraw {
 pub struct StakeEvent {
     /// Keys associated to the event.
     pub keys: StakeKeys,
-    /// Value of the relevant operation, be it `stake`, `unstake`,`withdraw`
+    /// Effective value of the relevant operation, be it `stake`,
+    /// `unstake`,`withdraw`
     pub value: u64,
+    /// The locked amount involved in the operation (e.g., for `stake` or
+    /// `unstake`). Defaults to zero for operations that do not involve
+    /// locking.
+    pub locked: u64,
```

Comment:
I would like some more context in the comment here:
What is the difference between `value` and `locked`?

Reply:
Locked contains a part of the original staked amount that has been moved (not eligibile anymore for consensus) as result of penalties (soft slash or topup).
Example:
1. User stake 10k dusk
2. User is soft slashed by 15% 
3. 1500 dusk are moved to locked
4. Current stake amount eligible becomes 8500 dusk

To recap, locked is just an amount that is excluded by the stake, but it still part of the stake amount sent from the transfer contract

This is the PR where the "locked" concept was introduced
https://github.com/dusk-network/rusk/pull/2232

Reply:
Thanks for the explanation! My comment was referring to fleshing out the comment of the field though.
Or if that's overkill at least link to a document where the `locked` amount of a stake is described in more detail.

Reply:
My bad, I didn't get it immediately

I'll try to include in the next round of rebase (if needed) otherwise we can add it as tech debt and write a followup issue